openid-sharedsignals-framework-1_0.md

```diff
@@ -829,6 +829,10 @@ an HTTP POST request to the Configuration Endpoint. On receiving a valid request
 the Event Transmitter responds with a "201 Created" response containing a
 [JSON][RFC7159] representation of the stream’s configuration in the body.
 
+If the stream already exists, the Event Transmitter responds with HTTP status code
```

Comment:
```suggestion
If a stream already exists, the expected behavior is for the Transmitter to create a new stream with a different ID. If the Transmitter does not want to allow multiple streams per receiver, it MAY respond with HTTP status code
```

Reply:
I agree with changing "the stream" to "a stream". However, instead of using "expected behavior is..." we should use the verbs "SHOULD" or "MUST" as appropriate. I feel having a "MUST" here will help interoperability. I suggest the following text: "If a stream already exists, and the Transmitter allows multiple streams with the same Receiver, then the Transmitter MUST respond with a new stream ID. If the Transmitter does not allow multiple streams with the same Receiver, it MUST respond with the HTTP status code 200 OK and include the representation of the existing stream's configuration".

Reply:
Addressed in [9497085](https://github.com/openid/sharedsignals/pull/50/commits/94970859e67adfb2248a79703986d356fdc2cb04)